rqcfilter.wdl

```diff
@@ -1,187 +1,42 @@
-workflow nmdc_rqcfilter {
-    String  container="bfoster1/img-omics:0.1.9"
+version 1.0
+import "shortReadsqc.wdl" as srqc
+import "longReadsqc.wdl" as lrqc
+
+workflow rqcfilter{
+    input{
+    # String  outdir
+    File?   reference
```

Comment:
We should be specifying the default paths to databases in the wdls, is there a reason this was changed to be specified in the input.json and as optional?